### PR TITLE
Moved package.json to config directory

### PR DESCRIPTION
**Pull request by TheBros35**
I think I made all the the correct changes - I could only find the helper.js that needed to be changed.

### CODE REVIEW

**Comment by TheBros35** [comment]
So I have some more changes - I added a dockerfile as per issue #2 and added a docker-compose example file as well. Please confer with csloz to ensure that my Dockerfile changes are correct if you choose to pull those commits in as well.

File: server/helpers.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const fs = require('fs')
 module.exports.writeDatabase = (data) => {
     return new Promise((resolve,reject) => {
 
-        fs.writeFile(`./database.json`, JSON.stringify(data), (error) => {
```
Comment (Enriquito) [inline server/helpers.js:6]: This path also needs to be changed to the new one.

File: server/helpers.js [hunk 2]
```diff
@@ -16,11 +16,11 @@ module.exports.writeDatabase = (data) => {
 }
 module.exports.readDatabase = () => {
     return new Promise((resolve,reject) => {
-        fs.readFile('./database.json', "utf8", function (error,data) {
```
Comment (Enriquito) [inline server/helpers.js:19]: This path also needs to be changed to the new one.

Comment (TheBros35) [reply server/helpers.js:19]: Sorry, I'm confused, that is what I changed - I changed the path of database.json that appears twice in this file to the new directory. Are there any other files / places besides those two that it appears in?

Comment (Enriquito) [reply server/helpers.js:19]: My bad. You are right!

**Comment by Enriquito** [comment]
@TheBros35 I am far from a expert on docker. So i have to trust you on this one. I will pull the update tonight.

**Pull request merged**